packages/ra-core/src/form/FormWithRedirect.tsx

```diff
@@ -150,17 +152,17 @@ const FormWithRedirect: FC<FormWithRedirectOwnProps & FormProps> = ({
 export interface FormWithRedirectOwnProps {
     defaultValue?: any;
     record?: Record;
-    save: (
+    save?: (
```

Comment:
We really should rename this to `onSubmit` imo

Reply:
that'd be a breaking change... Perhaps in a future major.

Reply:
Support both with a warning ?